pootle/apps/pootle_store/utils.py

```diff
@@ -212,17 +203,26 @@ def accept_suggestion(self, suggestion, update_unit):
         unit.target = suggestion.target
         if unit.state == FUZZY:
             unit.state = TRANSLATED
-
+        unit.submitted_by = suggestion.user
+        unit.submitted_on = current_time
+        unit.reviewed_by = self.reviewer
+        unit.reviewed_on = unit.submitted_on
+        unit.save(
+            submitted_by=suggestion.user,
+            submitted_on=current_time,
+            changed_with=self.review_type,
+            reviewed_by=self.reviewer,
+            reviewed_on=current_time)
         create_subs = OrderedDict()
         create_subs[SubmissionFields.TARGET] = [old_target, unit.target]
         if old_state != unit.state:
             create_subs[SubmissionFields.STATE] = [old_state, unit.state]
         subs_created = []
         for field in create_subs:
             kwargs = {
-                'creation_time': current_time,
-                'translation_project': translation_project,
-                'submitter': self.reviewer,
+                'creation_time': unit.change.reviewed_on,
+                'translation_project': unit.store.translation_project,
+                'submitter': suggestion.user,
```

Comment:
Submissions for accepting suggestions were previously credited to the reviewer, but now to the suggester. Is that ok?